sdk/appconfiguration/app-configuration/samples/getSettingOnlyIfChanged.ts

```diff
@@ -11,37 +11,50 @@ import { AppConfigurationClient } from "../src";
 
 export async function run() {
   console.log("Running get setting only if changed sample");
-    
+
   // You will need to set this environment variable
   const connectionString = process.env["AZ_CONFIG_CONNECTION"]!;
   const client = new AppConfigurationClient(connectionString);
 
   const key = "getSettingOnlyIfChangedExample";
   await cleanupSampleValues([key], client);
-  
+
   const addedSetting = await client.addConfigurationSetting({ key, value: "Initial value" });
 
   // now our application only wants to download the setting if it's changed
-  // when it's not changed we throw an error that you can inspect and deal 
+  // when it's not changed we throw an error that you can inspect and deal
```

Comment:
I guess this comment is no longer relevant?

Reply:
Oh man, yeah. 

I also updated another comment further down and codified the 304 convention so people are aware of it.